README.md

```diff
@@ -7,13 +7,13 @@ ESLint and prettier configuration for Fonto platform and projects.
 Install eslint and the Fonto configuration package:
 
 ```
-npm i -g babel-eslint eslint eslint-config-fontoxml eslint-config-prettier eslint-plugin-prettier eslint-plugin-react
+npm i -g babel-eslint eslint @fonto/eslint-config eslint-config-prettier eslint-plugin-prettier eslint-plugin-react
```

Comment:
Should this be `@fontoxml/eslint-config`?

Reply:
Yes it should.